### PR TITLE
76 add react i18next library and refactor texts

### PR DESCRIPTION
**Pull request by JamalAlabdullah**
Added react i18next library to save all texts in an own json file.

### CODE REVIEW

**Comment by JamalAlabdullah** [comment]
We have error in test because we need to mock the translation for all tests. I will add an issue for this

File: frontend/language/src/nor.json
Comment (ErlingHauan) [inline frontend/language/src/nor.json:1]: Filename is "nor.json", but text is in English.

File: frontend/language/src/nor.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+"form_builder": "Form Builder",
+"form_builder.form": "Form",
+"form_builder.drag.component.message": "Drag components here",
+"settings_side_bar": " Settings",
+"settings_side_bar.minimum.lenght": "Minimum length",
```
Comment (ErlingHauan) [inline frontend/language/src/nor.json:6]: Typo in key

File: frontend/language/src/nor.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+"settings_side_bar.maximum.lenght": "Maximum length",
```
Comment (ErlingHauan) [inline frontend/language/src/nor.json:7]: Typo in key

File: frontend/main-app/src/app/App.tsx
Comment (ErlingHauan) [inline frontend/main-app/src/app/App.tsx:1]: Language metadata should be changed to English 😊

File: frontend/main-app/src/app/App.test.tsx [hunk 3]
```diff
@@ -37,9 +37,9 @@ describe("App component", () => {
   });
 
   it("renders Form Builder when accessing '/form-builder'", () => {
-    renderApp(["/form-builder"]);
+    renderApp();
 
-    const title = screen.getByRole("heading", { name: "Form Builder" });
+    const title = screen.getByRole("link", { name: "Form Builder" });
```
Comment (ErlingHauan) [inline frontend/main-app/src/app/App.test.tsx:42]: Possible to use "name: header_form.factory.title.link"?

File: frontend/main-app/src/components/Login/Login.test.tsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { Login } from "./Login";
 import { MemoryRouter } from "react-router";
 
 describe("Login component", () => {
-  it("loads and displays Login component", () => {
+  it("loads and displays Login component", async () => {
```
Comment (ErlingHauan) [inline frontend/main-app/src/components/Login/Login.test.tsx:8]: I don't think async is required when using getByRole and getByLabelText
https://testing-library.com/docs/dom-testing-library/api-async/

File: frontend/main-app/src/components/Login/Login.tsx [hunk 5]
```diff
@@ -5,40 +5,52 @@ import React, { FormEvent, useState } from "react";
...
-        Log in
+        {t("login_page.title")}
       </Heading>
       <div className={classes.fieldContainer}>
-        <Textfield name="email" type="email" label="E-mail" error={formErrors?.email} />
-        <Textfield name="password" type="password" label="Password" error={formErrors?.password} />
+        <Textfield
+          name="email"
+          type="email"
+          label="signup_page.email.label"
```
Comment (ErlingHauan) [inline frontend/main-app/src/components/Login/Login.tsx:38]:
```suggestion
          label={t("signup_page.password.label")}
```

File: frontend/main-app/src/components/Login/Login.tsx [hunk 6]
```diff
@@ -5,40 +5,52 @@ import React, { FormEvent, useState } from "react";
...
+          error={formErrors?.email}
+        />
+        <Textfield
+          name="password"
+          type="password"
+          label="signup_page.password.label"
```
Comment (ErlingHauan) [inline frontend/main-app/src/components/Login/Login.tsx:44]:
```suggestion
          label={t("signup_page.password.label")}
```

**Review by ErlingHauan — CHANGES_REQUESTED** [review]
Nice work! 😊
I found some small things you can take a look at

File: frontend/language/src/nor.json
Comment (JamalAlabdullah) [reply frontend/language/src/nor.json:1]: Right,  we use english 👍😊

**Review by ErlingHauan — APPROVED** [review]
Good work! Very useful to have the texts sentralized in a JSON file 💯

**Pull request merged**